tutorial-5/src/mockservices/mod.rs

```diff
@@ -0,0 +1,13 @@
+pub mod rust;
+pub mod web3;
+pub mod world;
+
+pub use rust::RustService;
+pub use web3::Web3Service;
+pub use world::WorldService;
+
+use axum::response::{IntoResponse, Response};
+
+// pub trait Hello {
+//     fn hello(self) -> impl IntoResponse;
```

Comment:
trait's inner async function cannot be used for axum framework handler binding, give up define higher level traits.